Formula/weechat.rb

```diff
@@ -48,5 +47,7 @@ def install
 
   test do
     system "#{bin}/weechat", "-r", "/quit"
+    system "#{bin}/weechat", "-r", "/script install xterm_paste.pl; /wait 10 /quit"
```

Comment:
@ericboehs Still no joy here 😭. Would you mind trying this out locally and seeing what works for you? Shout if you need help with any of that.

Reply:
Didn't you forget `;` after wait ?

Reply:
That script looks right. Line 51 looks weird to me though. I'm not familiar with the ruby syntax that allows concatenating a variable with a string using just `/`. Maybe that's where the issue lines. 😕

Reply:
I did some troubleshooting. I came up with this:

```bash
brew uninstall --force weechat; brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/4bdc1ce82824246dfa49d002033e1d6b2fcabe81/Formula/weechat.rb --with-python@2 --with-perl --with-ruby --with-lua --with-aspell; rm ~/.weechat/perl/multiline.pl ~/.weechat/perl/autoload/multiline.pl 2> /dev/null; weechat -r "/script install multiline.pl; /wait 1 /quit"; ls ~/.weechat/perl/multiline.pl 2>&1 > /dev/null && echo "It worked" || echo "It failed"
```
This came back "It worked".

Then I ran
```bash
brew uninstall --force weechat; brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/4bdc1ce82824246dfa49d002033e1d6b2fcabe81/Formula/weechat.rb --with-python@2 --with-perl --with-ruby --with-lua --with-aspell --with-curl; rm ~/.weechat/perl/multiline.pl ~/.weechat/perl/autoload/multiline.pl 2> /dev/null; weechat -r "/script install multiline.pl; /wait 1 /quit"; ls ~/.weechat/perl/multiline.pl 2>&1 > /dev/null && echo "It worked" || echo "It failed"
```
This came back with "It failed".

Then I commented out the curl dependency via `brew edit weechat` and ran:

```bash
brew uninstall --force weechat; brew install weechat; rm ~/.weechat/perl/multiline.pl ~/.weechat/perl/autoload/multiline.pl 2> /dev/null; weechat -r "/script install multiline.pl; /wait 1 /quit"; ls ~/.weechat/perl/multiline.pl 2>&1 > /dev/null && echo "It worked" || echo "It failed"
```
And this too came back with "It failed". It doesn't seem to be strictly the curl dependency. Maybe it has something to do with compiling from source rather than pouring a bottle.

I'd suggest rolling back to 4bdc1ce to get the test to pass then slowly adding the changes from 692c152 until it fails.

Reply:
@ericboehs Even on 4bdc1ce I get the same test failure.